srcpkgs/ibus-libthai/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ibus-libthai'
+pkgname=ibus-libthai
+version=0.1.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="ibus-devel libthai gtk+3-devel"
+depends="ibus libthai gtk+3"
+short_desc="Libthai engine for IBus"
+maintainer="Kridsada Thanabulpong <sirn@ogsite.net>"
+license="GPL-2.0-or-later"
+homepage="https://linux.thai.net/projects/ibus-libthai"
+distfiles="http://linux.thai.net/pub/ThaiLinux/software/libthai/${pkgname}-${version}.tar.xz"
+checksum=fb71db057c293b5467e6fb96941969afe77da4e7852da578177df68cc0385545
+
+post_install() {
```

Comment:
Please remove this function. We already install the GPL system-wide so it's not necessary to explicitely include the license file with the package.